[dynamo] [easy] Move Set to dicts.py

lezcano opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #108420
* #110524
* #112252
* #111196
* #110523
* __->__ #110522

A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

lezcano committed:
[dynamo] Move Set to dicts.py

A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel reviewed (approved):
Assuming this is just code motion not changing anything

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo] [easy] Move Set to dicts.py"


A set is more of a dict than a list if you ask me.
This comes before the refactor where we implement sets and dicts via the
same logic.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]
